doc/output_config.md

```diff
@@ -41,6 +43,9 @@ the assumption that this is a non-overlapping list of particles. It is the users
 responsibility to make sure that this is the case. (See [known
 issues](#known-issues)).
 
+`ParticleFlowCandidate` collections from Delphes can also be stored as a `ReconstructedParticleCollection` in the output, but they currently have no associations to the generated particles.
```

Comment:
They will also not be added to the global `ReconstructedParticle` collection as that would lead to double counting (given that the `ParticleFlowCandidate` is a somewhat Delphes internal class)